### PR TITLE
resolve #1043 |  Enables Http header filtering

### LINKED ISSUE #1043: Header filters in Hermes-console
Currently JSON and Avro filters can be added via Hermes-console. Thanks to #753 we can also add Header filters via REST API. Unfortunately, Header filters aren’t supported via GUI. It would be nice to have this support in Hermes-console. Furthermore, it should be enabled/disabled by configuration as not all environments have to support Headers filters.

### PR DESCRIPTION
**Pull request by qrman**
This is how it looks on Subscription page:
Screenshot 2019-10-24 at 13 24 09

This is how it look in edit modal:
Screenshot 2019-10-24 at 13 24 22

### CODE REVIEW

File: hermes-console/static/partials/modal/editSubscription.html [hunk 1]
```diff
@@ -177,11 +177,12 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
             <div class="form-group">
                 <label class="col-md-3 control-label">Filter</label>
                 <div class="col-md-9">
-                    <table class="table borderless" ng-show="subscription.filters.length > 0">
+                    <table class="table borderless">
                         <tr>
                             <th>Type</th>
                             <th>Path</th>
                             <th>Matcher</th>
+                            <th>#</th>
```
Comment (dankraw) [inline hermes-console/static/partials/modal/editSubscription.html]: can we leave an empty header cell `<th></th>`?
`#` is usually used for rows enumeration.

Comment (qrman) [reply hermes-console/static/partials/modal/editSubscription.html]: of course, no problem.

Comment (dankraw) [reply hermes-console/static/partials/modal/editSubscription.html]: thanks :]

**Comment by qrman** [comment]
Should I do something more according to this part of an issue: `Furthermore, it should be enabled/disabled by configuration as not all environments have to support Headers filters.` ?

**Comment by cristaloleg** [comment]
@druminski ^^^

**Pull request merged**